sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -1,15 +1,13 @@
 <template>
   <full-page-container>
+    <p class="category-header-large primary-color">Search for application</p>
```

Comment:
this is `h1` 
![image](https://user-images.githubusercontent.com/77353155/196773654-0081ac7d-f384-47ad-8e3b-78f8e522813c.png)

Reply:
Please have a look on the wiki, it should give an idea
https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#components-guide-samples

Reply:
ok. Done. Thanks!

Reply:
Thanks! I'm using `body-header` now.